### PR TITLE
Public Cloud GKE scripts

### PR DESCRIPTION
**Pull request by bear454**

### CODE REVIEW

File: gke/main.tf [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring",
+    ]
+  }
+
+  management {
+    auto_repair  = false
+    auto_upgrade = false
+  }
+}
```
Comment (satadruroy) [inline gke/main.tf:70]: We might want to test with these two set to `true` - the reason I had these two set to `false` was to get around losing the swap accounting changes after updates but with swap accounting no longer being necessary we can test to see if CAP is functional even after an auto-upgrade/repair.

File: gke/scf-config-values.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    password: ""
+  organization: "cap"
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 1024m
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    cert-manager.io/acme-challenge-type: dns01
+    cert-manager.io/acme-dns01-provider: gcp-clouddns-provider
```
Comment (satadruroy) [inline gke/scf-config-values.yaml:28]: I think lines 27 and 28 are no longer required - can you test please without those?

https://cert-manager.io/docs/usage/ingress/#supported-annotations

Comment (bear454) [reply gke/scf-config-values.yaml]: Worked fine without; removed in a3e60a5.

File: gke/main.tf (hunk 1 above)
Comment (bear454) [reply gke/main.tf:70]: At this time, I don't see any good way to test this change, so I'd prefer to leave it as-is for release. I'd suggest filing an issue for it to be tested.

**Pull request merged**